Cotire uses precompiled headers which are more fragile with Clang.
For one, https://github.com/KhronosGroup/glslang/commit/e8c8af9c4b0d casues

error: POSIX thread support was disabled in PCH file but is currently enabled

--- rpcs3/CMakeLists.txt.orig	2019-05-31 16:49:32 UTC
+++ rpcs3/CMakeLists.txt
@@ -1,7 +1,5 @@
 cmake_minimum_required(VERSION 3.8.2)
 
-include(cotire)
-
 # Generate git-version.h at build time.
 include(${CMAKE_CURRENT_SOURCE_DIR}/git-version.cmake)
 
@@ -80,12 +78,6 @@ if(WIN32)
 else()
 	target_link_libraries(rpcs3 ${CMAKE_DL_LIBS})
 endif()
-
-set_target_properties(rpcs3 PROPERTIES
-	COTIRE_CXX_PREFIX_HEADER_INIT "${RPCS3_SRC_DIR}/stdafx.h"
-	COTIRE_ADD_UNITY_BUILD OFF)
-
-cotire(rpcs3)
 
 # Copy icons to executable directory
 if(APPLE)
--- rpcs3/Emu/CMakeLists.txt.orig	2019-05-31 16:49:32 UTC
+++ rpcs3/Emu/CMakeLists.txt
@@ -115,13 +115,3 @@ target_link_libraries(rpcs3_emu
 	PRIVATE
 		3rdparty::gsl 3rdparty::xxhash
 		3rdparty::dx12)
-
-
-# Setup cotire
-option(UNITY_BUILD_EMU "Use unity build for rpcs3_emu target" OFF)
-
-set_target_properties(rpcs3_emu PROPERTIES
-	COTIRE_CXX_PREFIX_HEADER_INIT "${RPCS3_SRC_DIR}/stdafx.h"
-	COTIRE_ADD_UNITY_BUILD ${UNITY_BUILD_EMU})
-
-cotire(rpcs3_emu)
